sockeye/rnn_attention.py

```diff
@@ -608,7 +666,8 @@ def on(self, source: mx.sym.Symbol, source_length: mx.sym.Symbol, source_seq_len
         :return: Attention callable.
         """
 
-        coverage_func = self.coverage.on(source, source_length, source_seq_len) if self.coverage else None
+        if self.coverage:
```

Comment:
explicit check for None

Reply:
don't you need to set coverage_func to None if self.coverage is None? I can imagine that the style checkers would complain about possibly uninitialized variables.

Reply:
no style check likes that :)